src/ShopBundle/objects/db/TShopModuleArticleListFilter/TShopModuleArticlelistFilterSearch.class.php

```diff
@@ -134,14 +140,20 @@ public function _AllowCache()
 
     /**
      * is called when the module initializes.
+     *
+     * @return void
      */
     public function ModuleInitHook()
     {
         parent::ModuleInitHook();
         $oShop = TdbShop::GetInstance();
         if ($oShop->fieldRedirectToNotFoundPageProductSearchOnNoResults) {
-            // in this case the shop may redirect after running the search. since this occurs in the Execute of the module, we need
-            // to prevent any other output from being auto-sent to the browser
+            /**
+             * in this case the shop may redirect after running the search. since this occurs in the Execute of the
+             * module, we need to prevent any other output from being auto-sent to the browser
+             * @psalm-suppress UndefinedInterfaceMethod
+             * @FIXME Method `SetBlockAutoFlushToBrowser` only exist on a single implementation of the interface
```

Comment:
Similar to the redirect method - `SetBlockAutoFlushToBrowser` only exists on a single implementation, not on the interface.